README.md

```diff
@@ -43,27 +41,6 @@ Running the Scorecard action on a fork repository is not supported.
 
 GitHub Enterprise repositories are not supported.
 
-## Personal Access Token (PAT) Requirements and Risks
```

Comment:
Done. I added it back (with a few modifications) near the end of the document, since it's no longer a part of the desired "flow". I also added a comment in the Authentication section that mentions classic tokens and directs readers to this section.